instrumentation/opentelemetry-instrumentation-grpc/tests/test_client_interceptor.py

```diff
@@ -41,13 +41,55 @@
 from .protobuf.test_server_pb2 import Request
 
 
+class Interceptor(
```

Comment:
Is this to test that the instrumentation doesn't break existing interceptors? Can we just add a comment explaining what this is for